sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetReadSupport.scala

```diff
@@ -322,6 +341,32 @@ private[parquet] object ParquetReadSupport {
     }
   }
 
+  /**
+   * Computes the structural intersection between two Parquet group types.
```

Comment:
It would be great that we mention this is because parquet library does not support the nested field access to non-existent field while parquet library does support to read the non-existent field by regular field access.

Reply:
I added like that.